src/sinks/influxdb/metrics.rs

```diff
@@ -95,13 +98,13 @@ impl InfluxDBSvc {
         let uri = settings.write_uri(endpoint)?;
 
         let build_request = move |body: Vec<u8>| {
-            let mut builder = hyper::Request::builder();
-            builder.method(Method::POST);
-            builder.uri(uri.clone());
-
-            builder.header("Content-Type", "text/plain");
-            builder.header("Authorization", format!("Token {}", token));
-            builder.body(body).unwrap()
+            hyper13::Request::builder()
+                .method(Method::POST)
+                .uri(uri.clone())
```

Comment:
- This is ok as in other PR, but can be shortened to `hyper13::Request::post(uri)` (in `mod.rs` this form used)
- No need clone `uri`

Reply:
True. Those are relics from older code.

Reply:
Ah, I see now. Skipped that closure `build_request` used in service.